### PR TITLE
ACD-4213: Updated the eventtype mapped CIM data model test case

### PR DESCRIPTION
**Pull request by rinkaldomadiya-crest**
- Created the fixture to get add-on associated all eventtype list
- Updated the eventtype mapped datamodel query by using the list of eventtype and datamodels tag.
- Verified the updated test case on multiple add-on.

### CODE REVIEW

**Comment by rfaircloth-splunk** [comment]
Can we add a failing test case to tests/?

**Comment by rinkaldomadiya-crest** [comment]
> Can we add a failing test case to tests/?

Added the failing test case

File: pytest_splunk_addon/standard_lib/cim_tests/test_templates.py [hunk 1]
```diff
@@ -247,57 +247,157 @@ def test_cim_fields_not_allowed_in_props(
...
-            "Web",
+            {"name": "Alerts", "tags": [["alert"]]},
+            {
+                "name": "Authentication",
+                "tags": [
+                    ["authentication"],
+                    ["authentication", "default"],
+                    ["authentication", "insecure"],
+                    ["authentication", "privileged"],
+                    ["authentication", "privileged"],
```
Comment (jayjoshi-crest) [inline pytest_splunk_addon/standard_lib/cim_tests/test_templates.py:275]: Can we remove duplicate tags: "authentication", "privileged"

Comment (rinkaldomadiya-crest) [reply pytest_splunk_addon/standard_lib/cim_tests/test_templates.py]: Removed

**Pull request merged**